mu-plugins/10up-plugin/package.json

```diff
@@ -15,20 +15,16 @@
     "node": ">=12.0.0"
   },
   "devDependencies": {
-    "10up-toolkit": "^2.1.0"
+    "10up-toolkit": "^2.2.0-alpha.8"
   },
   "dependencies": {
     "prop-types": "^15.7.2"
   },
   "10up-toolkit": {
-    "devURL": "https://my-project.test",
     "entry": {
       "admin": "./assets/js/admin/admin.js",
       "frontend": "./assets/js/frontend/frontend.js",
-      "shared": "./assets/js/shared/shared.js",
-      "admin-style": "./assets/css/admin/admin-style.css",
-      "shared-style": "./assets/css/shared/shared-style.css",
```

Comment:
I would love to better understand how the entry points now get generated. Is is depending on the name of the JS file they were imported into? Or is it depending on the name of the stylesheet that gets imported? 

There will be cases where a block for example has two separate stylesheets. One for the editor and one for the fronend. They should not be merged into the same stylesheet. 

I think `@wordpress/scripts` handles this by allowing for `style.css` or `editor.css`. And depending on which it is they get put into two separate files.

Additionally, it would be great for the next iteration of all this to allow for per block stylesheet generation. That way there could be a `style.css` and `editor.css` for every single block.

Reply:
The entrypoint gets generated based on the js file importing them.

So anything imported by `frontend.js` will get bundled in a `frontend.css` including block css etc.

I'm happy to expand this a bit further.

Reply:
Okay yeah, I think this is fine for now and makes sense. In the future when we want to tackle the individual stylesheets per block we will need to differentiate between editor and frontend styles. We can use `@wordpress/scripts` as an inspiration there.

Reply:
Sounds good.

We could expand this as part of the work to automatically detecting block.json files in 10up-toolkit